package.json

```diff
@@ -40,6 +38,9 @@
     "jest-junit": "^5.1.0",
     "snyk": "^1.168.0"
   },
+  "peerDependencies": {
+    "puppeteer": "18.1.0"
```

Comment:
**issue**: if the purpose of moving it to `peerDependencies` is to let consumers pick other versions, specifying an exact version here won't let them do that; `npm` will raise a peer dependency conflict if you try to install any other version at all

Reply:
Ahh ok. Thank you for clarifying.. the purpose was cos "that's the way we do it now" so your additional context makes more sense! I will update...

Reply:
there's two main reasons we use `peerDependencies`: letting consumers pick a different version (for things like Tool Kit's `webpack` plugin), or raising errors if your dependencies install different versions (for things like Origami components). this still might be useful to have for the first reason?